### PR TITLE
[MWPW-131024] Browse by category: Update design for template page V2

### PR DESCRIPTION
**Pull request by redvelvet511**
Fix [MWPW-131024] https://jira.corp.adobe.com/browse/MWPW-131024

**Description:**
- Added a variant (card) to browse by category. 
- Restructured image wrapper and image shadow. 
- Refactored CSS

**Test URLs:**
- Before: https://main--express-website--adobe.hlx.page/drafts/qiyundai/mwpw-131024/browse-by-category?lighthouse=on
- After: https://mwpw-131024--express-website--wbstry.hlx.page/drafts/qiyundai/mwpw-131024/browse-by-category?lighthouse=on

### CODE REVIEW

File: express/blocks/browse-by-category/browse-by-category.css [hunk 1]
```diff
@@ -1,150 +1,229 @@
...
+  justify-content: space-between;
+  flex-direction: column;
+  padding-left: 28px;
+  padding-right: 28px;
+  margin-left: auto;
+  margin-right: auto;
+  margin-bottom: 10px;
+}
+/* 
+main .browse-by-category.card .browse-by-category-category {
```
Comment (qiyundai) [inline express/blocks/browse-by-category/browse-by-category.css:29]: Shall we remove this altogether?

File: express/blocks/browse-by-category/browse-by-category.css [hunk 2]
```diff
@@ -1,150 +1,229 @@
...
+  flex-direction: column;
+  align-items: start;
+  min-width: 123px;
+  margin: 0;
+} */
+
+main
+  .browse-by-category
+  .browse-by-category-heading-section
+  .browse-by-category-heading {
```
Comment (qiyundai) [inline express/blocks/browse-by-category/browse-by-category.css:42]: Is your IDE doing this for you? I feel like we should stick to keeping the declaration on one line.

File: express/blocks/browse-by-category/browse-by-category.css [hunk 3]
```diff
@@ -1,150 +1,229 @@
...
+  border-color: var(--color-info-accent);
+  transform-origin: 75% 75%;
+  transform: rotate(-45deg);
+  content: "";
+  margin-top: 5px;
+  margin-left: 5px;
+  margin-right: 2.25px;
+}
+
+main .browse-by-category .browse-by-category-category {
```
Comment (qiyundai) [inline express/blocks/browse-by-category/browse-by-category.css:82]: It's my debt and nitpicking...do you mind renaming the .browse-by-category-category to something like .browse-by-category-card? and make sure you change the origin of it in browse-by-category.js as well! Thanks! 🙏

File: express/blocks/browse-by-category/browse-by-category.css [hunk 4]
```diff
@@ -1,150 +1,229 @@
...
+  display: flex;
+  gap: 8px;
+  flex-direction: column;
+  align-items: center;
+  min-width: 123px;
+  margin: 0;
+  padding: 10px 8px 0 8px;
+}
+
+main .browse-by-category .browse-by-category-category-anchor {
```
Comment (qiyundai) [inline express/blocks/browse-by-category/browse-by-category.css:93]: Same here, I think you can call it .browse-by-category-link

File: express/blocks/browse-by-category/browse-by-category.css [hunk 5]
```diff
@@ -1,150 +1,229 @@
...
+
+main
+  .browse-by-category
+  .browse-by-category-category-anchor:hover
+  ~ .browse-by-category-category-image-wrapper
+  img {
+  transform: scale(1.1) matrix(1, -0.07, 0.07, 1, 0, 0);
+}
+
+main .browse-by-category .browse-by-category-category-image-wrapper {
```
Comment (qiyundai) [inline express/blocks/browse-by-category/browse-by-category.css:107]: naming thing here too. You can just remove the extra "category"

File: express/blocks/browse-by-category/browse-by-category.css [hunk 6]
```diff
@@ -1,150 +1,229 @@
...
+  main .browse-by-category .browse-by-category-heading-section {
+    flex-direction: row;
+    padding: 0;
+  }
 
-    main .browse-by-category-container .browse-by-category-wrapper {
-        padding: 0 28px;
-        max-width: none;
-    }
+  main .browse-by-category .browse-by-category-wrapper {
```
Comment (qiyundai) [inline express/blocks/browse-by-category/browse-by-category.css:218]: I don't think this line is working. .browse-by-category is the block, the .browse-by-category-wrapper is outside the block. You can just do `main .browse-by-category-wrapper`

File: express/blocks/browse-by-category/browse-by-category.css [hunk 7]
```diff
@@ -1,150 +1,229 @@
...
+    padding: 0 28px;
+    max-width: none;
+  }
 }
 
 @media (min-width: 1200px) {
```
Comment (qiyundai) [inline express/blocks/browse-by-category/browse-by-category.css:224]: I think we can just get rid of this media query. The 900px one already covers it. I might have had other rules in here and forgot to delete it. Thanks!

File: express/blocks/browse-by-category/browse-by-category.js [hunk 8]
```diff
@@ -41,14 +41,16 @@ export function decorateCategories($block, payload) {
   payload.categories.forEach((category) => {
     const $category = createTag('div', { class: 'browse-by-category-category' });
     const $categoryImageWrapper = createTag('div', { class: 'browse-by-category-category-image-wrapper' });
+    const $categoryImageShadowWrapper = createTag('div', { class: 'browse-by-category-category-image-shadow-wrapper' });
     const $categoryImageShadow = createTag('div', { class: 'browse-by-category-category-image-shadow' });
     const $categoryImage = category.$image;
     const $categoryTitle = createTag('h4', { class: 'browse-by-category-category-title' });
     const $categoryAnchor = createTag('a', { class: 'browse-by-category-category-anchor' });
```
Comment (qiyundai) [inline express/blocks/browse-by-category/browse-by-category.js:48]: Same as in CSS, if you don't mind changing these names with duplicate 'category' that would be great! 🙏

**Review by qiyundai — CHANGES_REQUESTED** [review]
Looking good! Just a few more refactoring items. Really appreciate your work on this!

File: express/blocks/browse-by-category/browse-by-category.css (hunk 2 above)
Comment (redvelvet511) [reply express/blocks/browse-by-category/browse-by-category.css:42]: yep, the formatter, will fix it!

File: express/blocks/browse-by-category/browse-by-category.css (hunk 1 above)
Comment (redvelvet511) [reply express/blocks/browse-by-category/browse-by-category.css:29]: yep, will remove!

File: express/blocks/browse-by-category/browse-by-category.css (hunk 4 above)
Comment (redvelvet511) [reply express/blocks/browse-by-category/browse-by-category.css:93]: There's already an existing class called .browse-by-category-link that targets another specific link so I'll name this to .browse-by-category-card-link instead!

File: express/blocks/browse-by-category/browse-by-category.css (hunk 6 above)
Comment (redvelvet511) [reply express/blocks/browse-by-category/browse-by-category.css:218]: I checked and it's pretty much not doing anything, so I just removed it.

File: express/blocks/browse-by-category/browse-by-category.css (hunk 7 above)
Comment (redvelvet511) [reply express/blocks/browse-by-category/browse-by-category.css:224]: I added it to the CSS above before I removed it so it won't break!

**Review by qiyundai — APPROVED** [review]

**Review by JingleH — APPROVED** [review]

**Review by apganapa — APPROVED** [review]

**Pull request merged**